src/ocean/util/encode/Base64.d

```diff
@@ -507,6 +523,35 @@ unittest
         test!("==")(result, payload);
         result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==");
         test!("==")(result, payload);
+        result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw=");
+        test!("==")(result, payload);
+        result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw");
+        test!("==")(result, payload);
+    }
+
+    // single padding test
+    {
+        // try the string with the single `=` padding, and without it
+        auto result = cast(Const!(ubyte)[])"any carnal pleasure.";
+        test!("==")(result, decode("YW55IGNhcm5hbCBwbGVhc3VyZS4="));
+        test!("==")(result, decode("YW55IGNhcm5hbCBwbGVhc3VyZS4"));
+    }
+
+    // short decode test
+    {
+        auto result = cast(Const!(ubyte)[])"M";
+
+        test!("==")(result, decode("TQ"));
+        test!("==")(result, decode("TQ="));
+        test!("==")(result, decode("TQ=="));
+    }
+    // ditto
+    {
```

Comment:
It tests the case where you get less than four characters, and +/- padding. I'll set the documentation straight.